examples/images_contours_and_fields/plot_streamplot.py

```diff
@@ -61,6 +61,9 @@
 U[:20, :20] = np.nan
 U = np.ma.array(U, mask=mask)
 
+gs = gridspec.GridSpec(nrows=3, ncols=2, height_ratios=[1, 1, 2])
```

Comment:
Not sure this is the best way to do this. @jklymak?

Reply:
Yeah, don't create a new gridspec.  You can get the old one with `gs = axs.get_subpltospec().get_gridspec()`.